docs/gguf.md

```diff
@@ -18,6 +18,43 @@ GGUF is a format based on the existing GGJT, but makes a few changes to the form
 
 The key difference between GGJT and GGUF is the use of a key-value structure for the hyperparameters (now referred to as metadata), rather than a list of untyped values. This allows for new metadata to be added without breaking compatibility with existing models, and to annotate the model with additional information that may be useful for inference or for identifying the model.
 
+### GGUF Naming Convention
+
+GGUF follow a naming convention of `<Model>-<Version>-<ExpertsCount>x<Parameters>-<EncodingScheme>.gguf`
```

Comment:
@mofosyne great work!

Maybe the only missing information is to add the optional suffix about the shard info. 
Example: ["grok-1/grok-1-q4_0-00003-of-00009.gguf"](https://github.com/huggingface/huggingface.js/blob/d1c0cb8604db4f504fa33b233fc12b9a501abc17/packages/gguf/src/gguf.spec.ts#L243-L250)